CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/src/components/List @korvin89
+/src/components/unstable/Select @korvin89
```

Comment:
Should i suggest other components owners here? or we not going to merge this PR?

Reply:
Sure, you can add other components into this PR

Reply:
Initial author that i can find so far

Card @Lunory 
Skeleton @pseudopirate 
Table @mfyodorov 
Toaster @korvin89 
Tooltip @pseudopirate

Reply:
List @ma-efremoff

Reply:
and what about base components (Input, Radio, Checkbox etc)?